runtime/compiler/control/CompilationThread.cpp

```diff
@@ -13035,12 +13045,18 @@ queryJITServerAOTCacheFilter(const char *methodSig, TR::Method::Type ty, TR::Com
 bool
 TR::CompilationInfo::methodCanBeJITServerAOTCacheStored(const char *methodSig, TR::Method::Type ty)
    {
-   return queryJITServerAOTCacheFilter(methodSig, ty, TR::Options::_JITServerAOTCacheStoreFilters);
+   return queryJITServerFilter(methodSig, ty, TR::Options::_JITServerAOTCacheStoreFilters);
    }
 
 bool
 TR::CompilationInfo::methodCanBeJITServerAOTCacheLoaded(const char *methodSig, TR::Method::Type ty)
    {
-   return queryJITServerAOTCacheFilter(methodSig, ty, TR::Options::_JITServerAOTCacheLoadFilters);
+   return queryJITServerFilter(methodSig, ty, TR::Options::_JITServerAOTCacheLoadFilters);
    }
+
+bool
+TR::CompilationInfo::methodCanBeRemotelyCompiled(const char *methodSig, TR::Method::Type ty)
+   {
+   return queryJITServerFilter(methodSig, ty, TR::Options::_JITServerRemoteExcludeFilters);
+    }
```

Comment:
Small misalignment of ending curly bracket.

Reply:
Thanks Fixed.